scripts/update/Updater.php

```diff
@@ -61,5 +64,35 @@ public function update($initialVersion)
 
             $this->setVersion('0.2.0');
         }
+
+        if ($this->isVersion('0.2.0')) {
+            OntologyUpdater::syncModels();
+
+            $publishingService = $this->getServiceManager()->get(PublishingService::SERVICE_ID);
+            $actionOptions = $publishingService->getOption(PublishingService::OPTIONS_ACTIONS);
+            $actionOptions = array_merge($actionOptions, [
```

Comment:
```
PHP Warning:  array_merge(): Argument #1 is not an array in D:\domains\package-tao-basic\taoPublishing\scripts\update\Updater.php on line 74
```
Please check if `actionOptions` is array

Reply:
fixed